App/Components/ResourceCard.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from 'react'
+import Text from '../UI/Text'
+import { Button, Card } from 'react-native-paper'
+
+type Props = {|
+  resource: Object
+|}
+
+const ResourceCard = (props: Props) => {
+
+  const {resource} = props
+
+  const img_url = resource.image ? resource.image.url : 'img/placeholder.png'
+
+  return (
+    <Card style={{marginBottom: 10}}>
+      <Card.Cover
+        resizeMode={'contain'}
+        source={{uri: img_url}}/>
+      <Card.Title
+        title={resource.title}
+        subtitle={'Published Dec 5th 2019'}
+      />
+      <Card.Content>
+        <Text muted>{resource.description}</Text>
+      </Card.Content>
+      <Card.Actions>
+        <Button onPress={() => navigation.navigate('WebContent', {resource})}>
+          Learn More
+        </Button>
+      </Card.Actions>
+    </Card>
+  )
+}
+
+export default ResourceCard
```

Comment:
Always end with a new line
```suggestion
export default ResourceCard

```